### PR TITLE
[bitnami/node] Add verification tests for NodeJS container

### PR DESCRIPTION
**Pull request by mdhont**
### Description of the change

Adding VIB tests for NodeJS container.

### CODE REVIEW

File: .vib/node/goss/vars.yaml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  - npx
+  - node-gyp
+  - python
+  - yarn
+files:
+  - paths:
+      - /opt/bitnami/node/lib/node_modules/corepack/package.json
+      - /opt/bitnami/node/lib/node_modules/node-gyp/package.json
+      - /opt/bitnami/node/lib/node_modules/npm/package.json
+      - /opt/bitnami/node/lib/node_modules/yarn/package.json
```
Comment (joancafom) [inline .vib/node/goss/vars.yaml:14]: I don't think this is necessary. You are already checking the `binaries` and that they appear in `npm list -g`

File: .vib/node/goss/vars.yaml [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+directories:
+  - mode: "0755"
+    paths:
+      - /app
+      - /opt/bitnami/node/bin
+      - /opt/bitnami/node/lib/node_modules
+root_dir: /opt/bitnami
+modules:
+  - corepack
```
Comment (joancafom) [inline .vib/node/goss/vars.yaml:23]: corepack is installed but optional, I would not check it.

File: .vib/node/goss/vars.yaml [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+      - /opt/bitnami/node/lib/node_modules/corepack/package.json
+      - /opt/bitnami/node/lib/node_modules/node-gyp/package.json
+      - /opt/bitnami/node/lib/node_modules/npm/package.json
+      - /opt/bitnami/node/lib/node_modules/yarn/package.json
+directories:
+  - mode: "0755"
+    paths:
+      - /app
+      - /opt/bitnami/node/bin
+      - /opt/bitnami/node/lib/node_modules
```
Comment (FraPazGal) [inline .vib/node/goss/vars.yaml:20]: I would say you are already implicitly verifying these dirs exist by checking the different binaries and the list of modules.

**Review by FraPazGal — DISMISSED** [review]
LGTM, though remember to remove the VIB triggers from the Dockerfiles

**Review by joancafom — COMMENTED** [review]
As mentioned, you need to remove the changes performed to the Dockerfiles for this PR to be ready to merge.

This change will require re-approving the PR, so feel free to ping me when it's done 😁

**Review by joancafom — APPROVED** [review]

**Pull request merged**